ironfish/src/fileStores/config.ts

```diff
@@ -90,13 +90,24 @@ export type ConfigOptions = {
   /**
    * The number of CPU workers to use for long-running node operations, like creating
    * transactions and verifying blocks. 0 disables workers (this is likely to cause
-   * performance issues), and -1 auto-detects based on the number of CPU cores.
+   * performance issues), and a negative value resolves to the amount of parallelism units
+   * available to the process (as returned by `os.availableParallelism()`) minus one.
```

Comment:
While correct, node runners are often dev-ops / normal technical people, while this documentation is engineering focused. I think this is fine but I try not to refer to node documentation or something dev-ops people may struggle with.

Reply:
I didn't realize this was user-visible documentation; I'll correct that

Reply:
We usually copy these docs into the website, but it's manual and we don't need too. I think this is fine if we don't copy them.